save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -165,9 +165,14 @@ class WarnPlugin(
         }.asSequence()
     }
 
-    private fun String.makeThePathRelativeToTestRoot() =
-            this.replace("${testConfig.getRootConfig().directory}", "")
-                .trimStart('/', '\\')
+    private fun String.makeThePathRelativeToTestRoot(): String {
+        val testRoot = testConfig.getRootConfig().directory.toString()
+        if (testRoot == ".") {
```

Comment:
Bugfix in function in case, when test root dir == '.'

Before, for example for single file `Test1Test.kt` and test root dir == '.' we will get ``Test1Testkt``

However, it's temporary solution, since we already have almost the same function https://github.com/cqfn/save/issues/237